types/jasminewd2/index.d.ts

```diff
@@ -47,6 +47,14 @@ declare namespace jasmine {
     not: ArrayLikeMatchers<T>;
   }
 
+  // Add definition to be compatible with latest jasmine v3 types.
```

Comment:
@zvirja, could explain how this helps make jasminewd2 types compatible with jasmine v3 types?

Reply:
Sure! The reason `jasminewd2` is not compatible with the latest jasmine is the `FunctionMatchers<T>` interface I recently added to `jasmine`. Here in `jasminewd2` we are extending `Matchers<T>` interface, so because of that the `FunctionMatchers<T>` in `jasmine` library starts to incorrectly extend the `Matchers<T>` interface (the member with promise becomes missing) and TS linter complains about that.

As you can see around, we extend all the `jasmine` matchers inherited from `Matchers<T>`, e.g. `ArrayLikeMatchers<T>`, so my fix is not special.

Also `FunctionMatchers<T>` is not used for TS versions prior to 3.1, so interface will be there, but will not be actually used.